sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/AdministrationOperationsLiveTests.cs

```diff
@@ -46,43 +46,6 @@ public async Task GetResourceDetails()
             Assert.IsNotNull(resourceDetails.CustomDocumentModelLimit);
         }
 
-        [RecordedTest]
-        [TestCase(true)]
-        [TestCase(false)]
-        public async Task AdminOps(bool useTokenCredential)
```

Comment:
This test was a mess, testing three different scenarios at once. They've been split into tests `GetDocumentModel`, `GetDocumentModelThrowsWhenModelDoesNotExist`, and `GetDocumentModels`, all of them moved to the `DocumentModelAdministrationLiveTests` class.